macros/dune/source.sql

```diff
@@ -1,6 +1,6 @@
 {% macro source(source_name, table_name) %}
   {% set rel = builtins.source(source_name, table_name) %}
-  {%- if target.type == 'trino' -%}
+  {%- if target.type == 'trino' and source_name != 'dune' -%}
```

Comment:
i guess this macro change breaks the legacy check step?

Reply:
I think everything becomes `modified` which then triggers the legacy check for all unmigrated models?